modules/user/user.service.js

```diff
@@ -30,7 +39,16 @@ class UserService {
     });
 
     if (!checkedUser) {
-      const message = `User with provided ${[key]} not found`;
+      const message = [
```

Comment:
Please,  uppercase the name of the variable because it is the actual error message

Reply:
fixed